daemon/src/model/cfd.rs

```diff
@@ -354,14 +366,27 @@ pub struct SettlementProposal {
     pub maker: Amount,
 }
 
+/// Proposed collaborative settlement
+#[derive(Debug, Clone)]
+pub struct RollOverProposal {
+    pub order_id: OrderId,
+    pub timestamp: SystemTime,
+    pub term_length: TermLength,
+}
+
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub enum TermLength {
+    TwentyFourHours,
+}
```

Comment:
Do we have to have this or could we also just imply that we are rolling over by 24h? Having an enumeration here suggests that we introduce different term lengths soon. Is that the intention?

Extending the message with a field is not hard so I think I'd rather keep things lean here and not have unnecessary abstractions.

Reply:
> It's probably too early for having this configurable.
> I think we need to play around with different strategies but this can still be an implicit strategy.

It sounds like you agree! Are we deleting this enum then so the roll-over term is implicit?

Reply:
😂
Iremoved it in the force push.